ui-tests/src/test/java/org/openmrs/reference/RegisterPatientTest.java

```diff
@@ -33,10 +35,9 @@ public void tearDown() throws Exception {
         waitForPatientDeletion(patient.uuid);
     }
 
-    // Test for Story RA-71
     @Test
-    @Category(org.openmrs.reference.groups.BuildTests.class)
-    public void registerPatient() throws InterruptedException {
+    @Category(BuildTests.class)
+    public void registerPatientTest() throws InterruptedException {
```

Comment:
Any reason why you didnot remove interruptedException here ?

Reply:
> Any reason why you didnot remove interruptedException here ?

Made a response on this above.